src/main/java/Duke.java

```diff
@@ -1,10 +1,167 @@
+import java.util.Scanner;
+
 public class Duke {
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        greetWords();
+
+        Task[] tasks = new Task[100];
+        int countTask = 0;
+
+        while(true) {
+            Scanner in = new Scanner(System.in);
+            String words = in.nextLine();
+            printLine();
+
+            //exit the Duke
+            if (words.equals("bye")) {
+                System.out.println("     Bye. Hope to see you again soon!");
+                printLine();
+                return;
+            }
+
+            //list all tasks
+            else if (words.equals("list")) {
+                listTasks(tasks,countTask);
+            }
+
+            //Todo
+            else if (words.startsWith("todo")) {
+                if (words.length() <=5) {
+                    forgetName();
+                } else {
+                    //get the task description and type
+                    tasks[countTask] = new Todo(words.substring(5));
+
+                    addTask(tasks[countTask], countTask);
+                    countTask++;
+                }
+            }
+
+            //Deadline
+            else if (words.startsWith("deadline")) {
+                if (words.length() <= 9) {
```

Comment:
I think these numbers might be viewed as magic numbers

Reply:
oh ok, I forgot to change these numbers

Reply:
can I do some change to my code after the IP deadline?